Quoteau/Quoteau/NewQuote/DrawView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  DrawView.swift
+//  Quoteau
+//
+//  Created by Wiktor Górka on 18/12/2019.
+//  Copyright © 2019 Lunar Logic. All rights reserved.
+//
+
+import UIKit
+
+protocol NewLineDelgate {
+    func didAddNewLine(point: CGPoint)
+    func hideKeybord()
+}
+
+class DrawView: UIView {
+
+    var delegate: NewLineDelgate?
+    var drawingDisabled = false
```

Comment:
If you had a 'drawingEnabled' here a lot of guards would be more readable.
Instead of:
`guard drawingDisabled == false else { return }`
this: 
`guard drawingEnabled else { return }`